README.md

```diff
@@ -120,6 +120,52 @@ Client: Second Client
   - News for January 2013
 ```
 
+## Transactional
+
+Sample code that uses Transactional message detail and timeline endpoint API.
+```python
+from createsend import Transactional
+import os
+import sys
+
+auth = {'api_key': os.getenv('CREATESEND_API_KEY', '')}
+msg_id = os.getenv('MESSAGE_ID', '')
+
+if len(auth) == 0:
+    print("API Key Not Provided")
+    sys.exit(1)
+
+if len(msg_id) == 0:
+    print("Message ID Not Provided")
+    sys.exit(1)
+
+#auth = {'api_key': '[api_key]'}
+#msg_id = "[message id]" # e.g., becd8473-6a19-1feb-84c5-28d16948a5fc
+
+tx = Transactional(auth)
+
+# Get message details using message id
+msg_details = tx.message_details(msg_id, statistics=True, exclude_message_body=False)
```

Comment:
No need to set `statistics` to True if we are not reading opens/clicks in the example. 

Also, I would set `exclude_message_body`  to True and change the comment to something like:

`# Get message details using message id. We can optionally disable loading the body by setting exclude_message_body to True.`

Reply:
Makes sense @xitonix